verifier/src/report_body.rs

```diff
@@ -878,4 +900,117 @@ mod test {
             1
         );
     }
+
+    #[test]
+    fn attributes_verifier_display() {
+        let inner = Attributes::from(REPORT_BODY_SRC.attributes);
+        let verifier = AttributesVerifier::new(inner);
+        let display_string = format!("{}", verifier);
+
+        let expected = format!("The {} is {inner}", Attributes::spaced_struct_name());
+
+        assert_eq!(display_string, expected)
```

Comment:
:shrug: Not sure it's any better, it would require one to `use alloc::string::ToString`
```suggestion

        let expected = format!("The {} is {inner}", Attributes::spaced_struct_name());

        assert_eq!(verifier.to_string(), expected)
```

Reply:
Done.